tests/test_app.py

```
@@ -0,0 +1,90 @@
+    else:
+        return find_component_by_id(children, id_value)
+    return None
+
+
+def find_component_by_type(tree, comp_type):
+    # match based on class or by tag name fallback
+    try:
+        if tree.__class__ is comp_type:
+            return tree
+    except Exception:
+        pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)